mmv1/third_party/terraform/services/netapp/resource_netapp_flex_volume_test.go.erb

```diff
@@ -0,0 +1,104 @@
+<% autogen_exception -%>
```

Comment:
I suspect this file is largely redundant with `resource_netapp_volume_test.go`, so I would recommend dropping it, unless there is novel update behavior with FLEX level. Alternatively, you may want to consider a minimal example to show FLEX usage (which adds the documentation and test).

Reply:
removed the file